server/src/utils.rs

```diff
@@ -1,11 +1,7 @@
-use actix_web::HttpRequest;
 use serde::Deserialize;
 
-pub fn should_use_english(args: DetailsQueryArgs, req: HttpRequest) -> bool {
-    // we calculate the language from the request by checking if either the query or the cookie are set to en
-    let cookie_en = req.cookie("lang").map_or(false, |c| c.value() == "en");
-    let arg_en = args.lang.map_or(false, |c| c == "en");
-    arg_en || cookie_en
+pub fn should_use_english(args: DetailsQueryArgs) -> bool {
```

Comment:
We might want to consider to rename `DetailsQueryArgs` to something like `LangOption`
If we do it like this, we can make it more explicit, what is meant.

I also would think that we might want to change the implementation to a `impl DetailsQueryArgs` of said stuct, i.e. support a syntax like `args.should_use_english()`, as this is probably more ergonomic. 
What do you think?

Reply:
I would have to try this out, I'm not sure how to do this the best way in Rust. But if that's possible it would be more convenient.